src/main/scala/util/package.scala

```diff
@@ -234,23 +234,55 @@ package object util {
     map.view.map({ case (k, vs) => k -> vs.toList }).toList
   }
 
+/** provides operators useful for working with bidirectional [[Bundle]]s
+  * 
+  * In terms of [[Flipped]] with a producer 'p' and 'consumer' c:
+  * c :<= p // means drive all unflipped fields of 'c' from 'p' (e.g.: valid/bits)  
+  * c :=> p // means drive all flipped fields of 'c' from 'p' (e.g.: ready)  
+  * c :<> p // do both of the above
+  * p :<> c // do both of the above, but you'll probably get a Gender error later.
+  * 
+  * This utility class is needed because in [[chisel3]]:
+  * c := p // only works if there are no directions on fields.  
+  * c <> p // only works if one of those is an [[IO]] (not a [[Wire]]).
+  * 
+  * Compared with [[chisel3]] operators:
+  * c <> p   is an 'actual-direction'-inferred 'c :<> p' or 'p :<> c'
+  * c := p is equivalent to 'c :<= p' + 'c :=> p' or 'c:<> p'
```

Comment:
This is wrong.
c := p is equivalent to 'c :<= p' + 'p :=> c'.

'c :<> p' is not related to 'c := p' at all.

Reply:
In other words, 'c := p' says drive ALL fields of 'c' from 'p' regardless of their direction.

(which is pretty much only useful when you want to put a VIP monitor down that watches the wires of an existing producer-consumer pair)